app/assets/stylesheets/show.scss

```diff
@@ -109,10 +117,23 @@
           clear: both;
         }
       }
+
       .resource-files {
         background-color: $body-bg;
-        margin-left: 3rem;
-        width: 90%
+        margin-left: .5rem;
+        margin-right: .5rem;
+
+        tr {
+          td {
+            max-width: 0;
```

Comment:
@aaron-collier you resolved this, but it's still 0?

Reply:
@jcoyne apologies, I didn't realize that change didn't push (due to rebasing I think). Fixed.